core/src/main/resources/jenkins/security/ResourceDomainRecommendation/message.properties

```
@@ -0,0 +1,2 @@
+blurb = The default Content-Security-Policy is currently overridden using the <code>hudson.model.DirectoryBrowserSupport.CSP</code> system property, which is a potential security issue when browsing untrusted files. \
+  As an alternative, you can now set up a <strong>Resource Root URL</strong> that Jenkins will use to serve some static files without adding <code>Content-Security-Policy</code> headers.
```

not sure if `now` is needed then like that tend to get stale quickly
```suggestion
  As an alternative, you can set up a <strong>Resource Root URL</strong> that Jenkins will use to serve some static files without adding <code>Content-Security-Policy</code> headers.
```